js/projects.js

```diff
@@ -0,0 +1,168 @@
+import projectDatas from './projectDatas.js';
+
+document.querySelector('.card').innerHTML = projectDatas
+  .map(
+    (data) => `
+    <section class="${data.class}" id="portfolio">
+        <div class="project-img section-1">
+          <img class='projesct-one-img' src="${data.imageMob}" alt="work background">
+          <img class='image-dektop-1' src="${data.imageDesk}" alt="work background">
+        </div>
+
+      <div class="project-witeup section-1">
+          <div class="projectone-header">
+            <h2>${data.title}</h2>
+          </div>
+          <div class="projectone-header-Dk">
+            <h2>${data.title}</h2>
+          </div>
+
+      <div class='project-details-container'>
+            <ul class="project-details">
+              <li>
+                <h4 class="projectHeading">${data.canopy[0]}</h4>
+              </li>
```

Comment:
- [ ] Good job implementing the changes suggested by the previous code reviewer :clap: , however  I would like to suggest you to add something. Your `x-mark` button should be static when you scroll in the modal. Kindly make sure to adjust to increase the user :bust_in_silhouette:  experience :+1: 

![Screenshot from 2023-02-02 22-12-21](https://user-images.githubusercontent.com/80458199/216441344-6422778b-bc25-4593-a64d-467376b7ba6d.png)

Reply:
Thank you for reviewing my code
i will implement the suggestion soonet